### PR TITLE
update commons lib

### PR DESCRIPTION
**Pull request by beetlecrunch**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -58,6 +58,7 @@ pipeline {
         AWS = credentials('jenkins-aws')
         STORAGE_USR="${AWS_USR}"
         STORAGE_PSW="${AWS_PSW}"
+        MAIN_AUDIENCE = credentials('main-audience')
```
Comment (galales) [inline Jenkinsfile]: the audience is not a secret, I think it might be a committed constant in the pipeline

**Review by galales — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (beetlecrunch) [reply Jenkinsfile]: I use a Global Var in Jenkins.
Is it acceptable in your opinion?

**Review by ubaldop — APPROVED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -51,13 +51,14 @@ pipeline {
       }
     }
 
-    stage('Apply Kubernetes files') {
+    stage('Apply Kubernetes files') {z
```
Comment (galales) [inline Jenkinsfile]: `cat on the keyboard`

File: Jenkinsfile (hunk 1 above)
Comment (galales) [reply Jenkinsfile]: yup, works for me

**Pull request merged**